### ISSUE TITLE
jQuery 3 Typings

### ISSUE THREAD

**Issue by Ballpin** [DefinitelyTyped/DefinitelyTyped#15733]
jQuery 3 is missing in the repo.

I found the latest version here https://www.nuget.org/packages/jquery.TypeScript.DefinitelyTyped/ from DT.

How come it is not included in this git repo, Is the community trying to enforce us to use MS Package Manager now? -_-

**Comment by andy-ms** [comment]
Just the opposite, nuget publishing has been [deprecated](https://github.com/DefinitelyTyped/NugetAutomation#-update-).
The current definitions in `types/jquery/index.d.ts` are for versions 1 and 2. I don't think types for version 3 were ever written. You could help contribute by updating it for version 3.

**Comment by Ballpin** [comment]
@andy-ms I have never done it before so im not really sure how to set it up. I just started using TypeScript. 

On the link i sent i assumed that version 3 was already written but instead of putting it here "DefinitelyTyped" putted in nugget, in order to make ppl use Nugget instead through Visual Studio. The version that was being showed here in the repo was 2 years old while the one on the link i sent was updated half year ago.
